docs/tracing/create-tracing-app.md

```diff
@@ -0,0 +1,254 @@
+
+This guide walks you through the complete Tracing App creation process in Sematext Cloud, from initial setup through SDK configuration and agent installation.
+
+## Prerequisites
+
+- A Sematext Cloud account ([sign up free](https://apps.sematext.com/ui/registration))
+- Access to your application's source code or deployment configuration
+- Ability to install or access the Sematext Agent
+
+## Complete App Creation Flow
+
+### Step 1: Create Your Tracing App
+
+1.1 Navigate to App Creation:
+
+1. Log in to [Sematext Cloud](https://apps.sematext.com)
+2. Click "Create App" or "Create New Tracing App" in the main navigation
+3. Select "Tracing" as the App type
+
+1.2 Configure App Settings:
+
+![Create Tracing App](/docs/images/tracing/tracing-new-app-creation-1a.png)
+
+`App Name`:
+
+- Enter a descriptive name for your Tracing App
+- Use names that reflect your service or environment (e.g., "production-api", "user-service", "payment-gateway")
+
+`Plan Selection`:
+
+- Basic: Entry-level tracing with essential features
+- Standard: Enhanced capabilities with longer data retention  
+- Pro: Full-featured tracing with advanced analytics
+
+`Infra App Selection`:
+
+- Select existing Infra App: Choose from available Infra Apps if you have any
+- Create new Infra App: If none exists, a new one will be automatically created
+- The Infra App is required for agent communication and will be configured for tracing
+
+### Step 2: Configure OpenTelemetry SDK
+
+After App creation, the wizard continues with SDK configuration:
+
+![SDK Selection](/docs/images/tracing/tracing-new-app-creation-2.png)
+
+2.1 Select Your Programming Language.
+
+Choose from the available programming languages:
+
+- Java (including Kotlin)
+- Python 
+- Node.js
+- Go
+- .NET
+- Ruby
+- PHP
+- JavaScript
+
+2.2 Configure Service Names and Follow Instructions:
+
+![Service Configuration](/docs/images/tracing/tracing-new-app-creation-3a.png)
+
+Name the Service:
+
+- Enter a meaningful name for your application service
+- This should match your actual service name in your application
+
+Follow SDK Instructions:
+The UI will provide complete, language-specific instructions including:
+
+- Required dependencies and installation commands
+- Code examples for auto-instrumentation (recommended)
+- Manual instrumentation examples (advanced)
+- Environment variables and configuration options
+- Framework-specific guidance
+
+Add Multiple Services (Optional):
+
+- Click "Add Another Service" to configure multiple services at once
+- Each service will have its own configuration and will appear in the service list
+
+### Step 3: Install and Configure Sematext Agent
+
+The wizard continues with agent installation options:
+
+![Agent Installation Options](/docs/images/tracing/tracing-new-app-creation-4a.png)
+
+3.1 Choose Deployment Method:
+
+Select from four deployment methods:
+
+Single Host:
+
+- Best for: Development environments, single servers, or testing
```

Comment:
It's also good for production if you have e.g. 10 servers and you want to install the agent on each of them individually. We don't want people to think this is for non-prod.